src/bp/core/services/ghost/service.ts

```diff
@@ -16,10 +16,12 @@ import { VError } from 'verror'
 import { createArchive } from '../../misc/archive'
 import { TYPES } from '../../types'
 
-import { PendingRevisions, ReplaceContent, ServerWidePendingRevisions, StorageDriver } from '.'
+import { PendingRevisions, ReplaceContent, ServerWidePendingRevisions, StorageDriver, FileRevision } from '.'
 import DBStorageDriver from './db-driver'
 import DiskStorageDriver from './disk-driver'
 
+export type FileChanges = { scope: string; local: string[]; prod: string[] }[]
```

Comment:
I mean remote is not necessarily  remote either. It could be ghost though.

Reply:
Also the ghost was previously enabled when bp_production=true so it makes sense to call the "ghosted files" production files.

Reply:
I know we used bp_production before, but it wasn't making much sense either. It would make more sense to treat files as local on your disk, and remote in the database